### DISCUSSION TITLE
Модули ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#73] [Announcements]
# Модули ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/metanit/modules/

**Comment by nikyspace** [comment]
Почему на if (hour > 16) на знаке '>' выдает ошибку;
Также в следующем выражении выдает ошибку на "else"
??
